test/widgets/login_test.dart

```diff
@@ -61,7 +64,48 @@ void main() {
     expectErrorFromText('email@example.com', ServerUrlValidationError.noUseEmail);
   });
 
-  // TODO test AddAccountPage
+  group('AddAccountPage Server URL Helper Text', () {
+    Future<void> prepareAddAccountPage(WidgetTester tester) async {
+      await tester.pumpWidget(const MaterialApp(
+        localizationsDelegates: ZulipLocalizations.localizationsDelegates,
+        supportedLocales: ZulipLocalizations.supportedLocales,
+        home: AddAccountPage(),
+      ));
+    }
+
+    final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
+
+    Future<Finder> findHelperText(WidgetTester tester) async {
+      return find.text(zulipLocalizations.serverUrlDocLinkLabel);
+    }
+
+    testWidgets('launches URL when helper text is tapped', (tester) async {
+      await prepareAddAccountPage(tester);
+      final helper = await findHelperText(tester);
+      await tester.tap(helper);
+
+      check(testBinding.takeLaunchUrlCalls()).single.equals((
+        url: AddAccountPage.serverUrlHelpUrl,
+        mode: LaunchMode.platformDefault,
+      ));
+    });
+
+    testWidgets('shows error dialog when URL fails to open', (tester) async {
+      await prepareAddAccountPage(tester);
+      testBinding.launchUrlResult = false;
+      final helper = await findHelperText(tester);
+      await tester.tap(helper);
+      await tester.pump();
+
+      await tester.tap(find.byWidget(checkErrorDialog(
+        tester,
+        expectedTitle: zulipLocalizations.errorUnableToOpenLinkTitle,
+        expectedMessage: zulipLocalizations.errorUnableToOpenLinkMessage(
+          AddAccountPage.serverUrlHelpUrl.toString(),
+        ),
+      )));
```

Comment:
```suggestion
      await tester.tap(find.byWidget(checkErrorDialog(tester,
        expectedTitle: zulipLocalizations.errorUnableToOpenLinkTitle,
        expectedMessage: zulipLocalizations.errorUnableToOpenLinkMessage(
          AddAccountPage.serverUrlHelpUrl.toString(),
        ))));
```

nit: could be denser here, using fewer lines